data/importers/page.md

```diff
@@ -185,39 +185,187 @@ For each importer, we will cover only columns of interest.
+*Note: Can be blank, but the column is required*  
 ---
 * Arrival Stand Group
-* Stand Group ID can be retrieved from Export. This field can be left blank but the column is required
+* Stand Group ID can be retrieved from Export.  
+*Note: Can be blank, but the column is required*
 ---
-* Tag
-* Comma separated list of tags - Summer 2024, Winter 2024. This field can be left blank but the column is required.
+* PAX LF
+* A single integer containing relevant PAX Load Factor ID. The ID can be retrieved from VDS or Load Factor export.  
+*Note: Can be blank, but the column is required*
+---
+* Luggage LF
+* A single integer containing relevant Luggage Load Factor ID. The ID can be retrieved from VDS or Load Factor export.  
+*Note: Can be blank, but the column is required*
+---
+* Cargo (Weight) LF
+* A single integer containing relevant Cargo Weight Load Factor ID. The ID can be retrieved from VDS or Load Factor export.  
+*Note: Can be blank, but the column is required*
+---
+* Cargo (Volume) LF
+* A single integer containing relevant Cargo Volume Load Factor ID. The ID can be retrieved from VDS or Load Factor export.  
+*Note: Can be blank, but the column is required*
+---
+* Containers
+* Comma separated IDs of relevant container types. The IDs can be retrieved from VDS or Containers export.  
+*Note: Can be blank, but the column is required*
+---
+* Flightrules
+* Simbrief API value for ICAO Flight Plan - Flight Rules field. Can be "i" or "v".  
+*Note: Can be blank, but the column is required. Find out more on the Simbrief API description at the Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298*
+---
+* Flighttype
+* Simbrief API value for ICAO Flight Plan - Flight Type field. Can be "s", "n", "g", "m","x".  
+*Note: Can be blank, but the column is required. Find out more on the Simbrief API description at the Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298*
+---
+* Contpct
+* Simbrief API value for Contingency Fuel in percents (0.05), or percents/minutes (0.05/15).  
+*Note: Can be blank, but the column is required. Find out more on the Simbrief API description at the Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298*
+---
+* Resvrule
+* Simbrief API value for Reserve Fuel in minutes.  
+*Note: Can be blank, but the column is required. Find out more on the Simbrief API description at the Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298*
+---
+* Taxifuel
+* Simbrief API value for Taxi Fuel in minutes.  
+*Note: Can be blank, but the column is required. Find out more on the Simbrief API description at the Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298*
+---
+* Minfob
+* Simbrief API value for the Minimum Block fuel in Pounds.  
+*Note: Can be blank, but the column is required. Find out more on the Simbrief API description at the Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298*
```

Comment:
Navigraph Forum - https://forum.navigraph.com/t/the-simbrief-api/5298
should instead be
`[Navigraph Forum](https://forum.navigraph.com/t/the-simbrief-api/5298)`
to create a nice hyperlink